Libraries/pbxbuild/Sources/Build/DependencyResolver.cpp

```diff
@@ -306,14 +306,13 @@ resolveSchemeDependencies(Build::Context const &context) const
             }
         }
 
-        DependenciesContext dependenciesContext = {
-            .buildEnvironment = &_buildEnvironment,
-            .buildContext     = &context,
-            .graph = &graph,
-            .buildAction = buildAction,
-            .positional = &positional,
-            .productNameToTarget = &productNameToTarget,
-        };
+        DependenciesContext dependenciesContext;
+        dependenciesContext.buildEnvironment = &_buildEnvironment;
```

Comment:
This struct is complicated enough I figured it was worth having the names inline rather than guessing at the order.